src/builder.rs

```diff
@@ -0,0 +1,149 @@
+use crate::{
+    config::Config,
+    value::Value,
+    source::Source,
+    error,
+    error::ConfigError,
+    path::Expression
+};
+use std::collections::HashMap;
```

Comment:
Please import from `std` first (you'll need a blank line between the two import blocks).

Reply:
Done :)